zendesk/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: zendesk
 title: Zendesk
 description: Regulate customer communications by managing tickets and users in Zendesk data
-version: 2.0.1
+version: 2.0.2
```

Comment:
This should be bumped to 3.0.0 since the output for the Search action has changed backwards-incompatibly (for the better :) )

Reply:
@mrinehart-r7 Bumped :)